views/main/play.ejs

```diff
@@ -9,66 +9,72 @@
         alt="pixelated image"
       />
     </div>
-    <div id="guess-div">
-      <h5 class="card-title">What form of game is this?</h5>
-      <div class="field">
-        <fieldset>
-          <div>
-            <input
-              class="field-input"
-              type="radio"
-              id="a"
-              name="guess"
-              value="<%-gameName%>"
-            />
-            <label for="a"><%-gameName%></label>
-          </div>
-          <div>
-            <input
-              class="field-input"
-              type="radio"
-              id="b"
-              name="guess"
-              value="b"
-            />
-            <label for="b">b</label>
-          </div>
-          <div>
-            <input
-              class="field-input"
-              type="radio"
-              id="c"
-              name="guess"
-              value="c"
-            />
-            <label for="c">c</label>
-          </div>
-          <div>
-            <input
-              class="field-input"
-              type="radio"
-              id="d"
-              name="guess"
-              value="d"
-            />
-            <label for="d">d</label>
-          </div>
-        </fieldset>
+    <form action="/update-play-data" method="POST">
+      <div id="guess-div">
+        <h5 class="card-title">What form of game is this?</h5>
+        <div class="field">
+          <fieldset>
+            <div>
+              <input
+                class="field-input"
+                type="radio"
+                id="a"
```

Comment:
Since I commented on the other PR about descriptive id's, I think you should change it here too